fbw-a380x/src/systems/instruments/src/PFD/LowerArea.tsx

```diff
@@ -218,9 +218,9 @@ class FlapsIndicator extends DisplayComponent<{ bus: EventBus }> {
                     <path d={circlePath(0.8, 5, 196.4)} class={this.slatsTargetPos.map((i) => (i == 2 ? 'NormalStroke Stroke Fill Cyan' : 'NormalStroke White'))} />
 
                     <path d='M 32.3 193.7 v 1.7 h 1.9 z' class='Fill Stroke NormalStroke Cyan CornerRound' visibility={this.flapsTargetPos.map((i) => (i == 0 ? 'visible' : 'hidden'))} />
-                    <path d='M 39.9 196.8 v 1.7 h 1.9 z' class={this.flapsTargetPos.map((i) => (i == 1 ? 'Fill Stroke NormalStroke Cyan CornerRound' : 'Fill Stroke NormalStroke White CornerRound'))} />
-                    <path d='M 47.3 199.9 v 1.7 h 1.9 z' class={this.flapsTargetPos.map((i) => (i == 2 ? 'Fill Stroke NormalStroke Cyan CornerRound' : 'Fill Stroke NormalStroke White CornerRound'))} />
-                    <path d='M 54.7 203 v 1.7 h 1.9 z' class={this.flapsTargetPos.map((i) => (i == 3 ? 'Fill Stroke NormalStroke Cyan CornerRound' : 'Fill Stroke NormalStroke White CornerRound'))} />
+                    <path d='M 41.5 197.5 v 1.7 h 1.9 z' class={this.flapsTargetPos.map((i) => (i == 1 ? 'Fill Stroke NormalStroke Cyan CornerRound' : 'Fill Stroke NormalStroke White CornerRound'))} />
```

Comment:
We should not edit the flap/slat dots, they should be correct for the correct flaps deflections, we currently have the 320 deflections still, so they should match up once that has been fixed

Reply:
Ok, reverted these changes.